src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,374 @@
+@import "../../patternfly-utilities";
+
+.pf-c-sequence {
+  --pf-c-sequence--BackgroundColor: var(--pf-global--BackgroundColor--dark-100); // this background color is interchangable
+
+  // Sequence Control
+  --pf-c-sequence__control--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderTopColor: transparent;
+  --pf-c-sequence__control--BorderBottomColor: transparent;
+  --pf-c-sequence__control--hover--BorderTopHeight: var(--pf-global--BorderWidth--lg);
```

Comment:
technically speaking, the `Border...Height` vars should be `Border...Width` since `border-width` is the property you're modifying.

Reply:
replaced all Border...Height's with Border...Width